labelbox/schema/model_run.py

```diff
@@ -122,6 +125,34 @@ def _wait_until_done(self, status_fn, timeout_seconds=120, sleep_time=5):
                 )
             time.sleep(sleep_time)
 
+    def upsert_predictions_and_send_to_project(
+        self,
+        name: str,
+        predictions: LabelList,
+        project_id: str,
+        priority: Optional[int] = 5,
+    ) -> 'MEAPredictionImport':  # type: ignore
+        """ Upload predictions and creates a batch import to project.
+        Args:
+            name (str): name of the AnnotationImport job as well as the name of the batch import
+            predictions (Iterable):
+                iterable of annotation rows
+            project_id (str): id of the project to import into
+            priority (int): priority of the job
+        Returns:
+            (AnnotationImport, Project)
+        """
+        data_rows_set = set(
+            map(lambda x: x.data_row.id, predictions)[:DATAROWS_IMPORT_LIMIT])
+        data_rows = list(data_rows_set)
+        project = self.client.get_project(project_id)
+        batch = project.create_batch(name, data_rows, priority)
+
+        predictions_for_data_rows = filter(
+            lambda x: x.data_row.id in data_rows_set, predictions)
+
+        return (self.add_predictions(name, predictions_for_data_rows), batch)
```

Comment:
This never actually sends the data to the project. There needs to be a another step for invoking the MEA to MAL workflow.